themes/finna2/templates/RecordDriver/SolrEad/result-condensed.phtml

```diff
@@ -71,9 +71,11 @@
     <?php endif; ?>
     <div class="media-body">
       <div class="result-body">
-        <?php if ($origination = $this->driver->getOrigination()): ?>
+        <?php if ($originations = $this->driver->tryMethod('getOriginations', [], $this->driver->getOrigination())): $originations = (array)$originations; ?>
             <span class="archive-label"><?=$this->transEsc('Archive Origination')?>:</span>
-            <a href="<?=$this->url('search-results')?>?lookfor=%22<?=urlencode($origination)?>%22&amp;type=Author"><?=$this->escapeHtml($origination)?></a>
+            <?php $originationCount = count($originations); foreach ($originations as $i => $origination): ?>
+            <a href="<?=$this->url('search-results')?>?lookfor=%22<?=urlencode($origination)?>%22&amp;type=Author"><?=$this->escapeHtml($origination)?></a><?=$i + 1 < $originationCount ? ' ; ' : ''?>
```

Comment:
Kun näitä nyt viilataan, niin olisi hyvä käyttää url-helperiä parametrien lisäämiseen ja ajaa tulos escapeHtmlAttr:n kautta. Ks. esim. https://github.com/NatLibFi/NDL-VuFind2/blob/dev/themes/finna2/templates/librarycards/home.phtml#L79